src/client/index.tsx

```diff
@@ -5,72 +5,62 @@ import ReactDOM from "react-dom";
 const name = "John Doe";
 
 const consultantElement: JSX.Element = (
-    <div className={"divTable"}>
-        <div className={"headRow"}>
-            <div className={"divCell"}>{"Consultant"}</div>
-        </div>
-        <div className={"divRow"}>
-            <div className={"divCell"}>{"Name"}</div>
-            <div className={"divCell"}>{name}</div>
-        </div>
-        <div className={"divRow"}>
-            <div className={"divCell"}>{"Phone"}</div>
-            <div className={"divCell"}>{"1111111"}</div>
-        </div>
-        <div className={"divRow"}>
-            <div className={"divCell"}>{"Email"}</div>
-            <div className={"divCell"}>{"John@doe.com"}</div>
-        </div>
-        <div className={"divRow"}>
-            <div className={"divCell"}>{"Address"}</div>
-            <div className={"divCell"}>{"Accra, Ghana"}</div>
-        </div>
-        <div className={"divRow"}>
-            <div className={"divCell"}>{""}</div>
-            <div className={"divCell"}>{"2771 John st."}</div>
-        </div>
+    <div className="Rtable Rtable--2cols">
```

Comment:
Please wrap up property values in `{}`. Because in the future we will have different value types, and most of them won't work without `{}`, so it's better to keep same style for everything. Examples:
```
prop={{ test: "something }}
prop={null}
prop={[ "test" ]}
prop={"test"}
```
From everything above, only `string` works without `{}`. But if you use them everywhere, brain will just ignore one layer of `{}`, and it'll be easier to read.